lib/modules/manager/helmfile/extract.ts

```diff
@@ -1,4 +1,5 @@
 import is from '@sindresorhus/is';
+import urlJoin from 'url-join';
```

Comment:
use our own abstraction
```suggestion
import { joinUrlParts } from '../../../util/url';
```